### PR TITLE
Add oc new-app example to the README

### PR DESCRIPTION
**Pull request by ldimaggi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,20 @@ https://example-jenkins-jenkins-demo.apps.my-cluster.testcluster.openshift.com:4
...
+
+## Creating a Pipeline
+
+Creating a new project that defines a pipeline (for example, through a
+project's JenkinsFile) will automatically create a job and build in
+the Operator managed instance of Jenkins.
+
+For example:
+``` bash
+oc new-app https://github.com/waveywaves/nodejs-ex --context-dir openshift/pipelines
```
Comment (sbose78) [inline README.md]: Interesting, does this really use the operator managed jenkins?

Comment (akram) [reply README.md]: this uses any installed jenkins with the service name "jenkins". The readme update is useful for users who want to simply verify that the Jenkins is up and running and runnable.

**Pull request merged**